api/envoy/extensions/access_loggers/stderr/v3/stderr.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package envoy.extensions.access_loggers.stderr.v3;
+
+import "envoy/config/core/v3/substitution_format_string.proto";
+
+import "google/protobuf/struct.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.access_loggers.stderr.v3";
+option java_outer_classname = "StderrProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Standard Error access log]
+// [#extension: envoy.access_loggers.stderr]
+
+// Custom configuration for an :ref:`AccessLog <envoy_api_msg_config.accesslog.v3.AccessLog>`
+// that writes log entries directly to a standard error.
```

Comment:
```suggestion
// that writes log entries directly to the operating system's standard error.
```

? Similar edit below?

Reply:
fixed in 7b221f7aabfb02994e96528dfa648421e1f54a7b